javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/EndpointFeatures.qll

```diff
@@ -145,6 +159,65 @@ module FunctionBodies {
   }
 }
 
+/** This module provides functionality for getting the local neighborhood around an AST node within its enclosing function body, providing a locally-scoped version of the `enclosingFunctionBody` feature. */
+module NeighborhoodBodies {
```

Comment:
Copying this over from https://github.com/github/codeql/pull/7148 so it doesn't get lost in the shuffle:

Can the constants 128 and 256 be pulled out and passed as parameters? We may want to add several features with different values, to cover narrower and broader scopes.

Reply:
Done in 60ef70726c3a614fa469267423c3d47013714e81